packages/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -77,10 +77,10 @@ type SliderCommons = {
   disabled?: boolean;
 
   /**
-   * Whether to render the Slider vertically.
-   * @default `false` (renders horizontally)
+   * How to visually render the slider.
+   * @default `horizontal`
    */
-  vertical?: boolean;
+  appearance?: 'vertical' | 'horizontal';
```

Comment:
I think this should be called `orientation`, since `appearance` is used in other components with a different meaning (outline/underline/filled).